common/lib/client/auth.js

```diff
@@ -260,7 +259,13 @@ var Auth = (function() {
 						}
 					}
 					cb(null, body);
-				});
+				};
+				Logger.logAction(Logger.LOG_MICRO, 'Auth.requestToken().tokenRequestCallback', 'Sending; ' + authOptions.authUrl + '; Params: ' + JSON.stringify(authParams));
+				if(authOptions.authMethod && authOptions.authMethod.toLowerCase() === 'post') {
+					Http.postUri(rest, authOptions.authUrl, authHeaders || {}, authParams, {}, authUrlRequestCallback);
```

Comment:
(though on re-reading it, spec also says it should be "form-encoded", ably-js json-encodes. does 'form-encoding' just mean URI querystring encoding? If so surely that can't encode arbitrary javascript objects, which `authParams` will be... Not sure if I'm missing something)